test/controllers/orders_controller_test.rb

```diff
@@ -1,7 +1,13 @@
 require "test_helper"
 
 describe OrdersController do
-  # it "does a thing" do
-  #   value(1+1).must_equal 2
-  # end
+  
+  describe "validation - logged in" do
```

Comment:
I think this is going to conflict with Georgina's if we merge to master.

```suggestion
  describe "validation - logged in" do
  it "is valid with an order_item" do
   end
```